docs/Configuration.md

```diff
@@ -244,227 +395,151 @@ The JsonPathContentTemplate allows matching on and extracting values from an Eve
-                "valueExpression": "$.heartRate",
+                "valueExpression": "$.matchedToken.heartRate",
                 "valueName": "hr"
             }
         ]
     }
 }
 ```
 
-### **IotJsonPathContentTemplate**
-
-The IotJsonPathContentTemplate is similar to the JsonPathContentTemplate except the DeviceIdExpression and TimestampExpression are not required.
-
-The assumption when using this template is the messages being evaluated were sent using the [Azure IoT Hub Device SDKs](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks#azure-iot-hub-device-sdks) or [Export Data (legacy)](https://docs.microsoft.com/en-us/azure/iot-central/core/howto-export-data-legacy) feature of [Azure IoT Central](https://docs.microsoft.com/en-us/azure/iot-central/core/howto-export-data). When using these SDKs the device identity (assuming the device id from Iot Hub/Central is registered as an identifer for a device resource on the destination FHIR server) is known as well as the timestamp of the message.  If you are using Azure IoT Hub Device SDKs but are using custom properties in the message body for the device identity or measurement timestamp you can still use the JsonPathContentTemplate.
-
-*Note: When using the IotJsonPathContentTemplate the TypeMatchExpression should resolve to the entire message as a JToken.  Please see the examples below.*
-
-#### Examples
-
 ---
 
-**Heart Rate**
+**Project Data From Matched Token And Original Event**
 
 *Message*
+
 ```json
 {
-    "Body": {
-        "heartRate": "78"
-    },
+    "Body": [
+        {
+            "heartRate": "78",
+            "endDate": "2019-02-01T22:46:01.8750000Z",
+        },
+        {
+            "heartRate": "81",
+            "endDate": "2019-02-01T23:46:01.8750000Z",
+        },
+        {
+            "heartRate": "72",
+            "endDate": "2019-02-01T24:46:01.8750000Z",
+        }
+    ],
     "Properties": {
-        "iothub-creation-time-utc" : "2019-02-01T22:46:01.8750000Z"
+        "deviceId": "device123"
     },
-    "SystemProperties": {
-        "iothub-connection-device-id" : "device123"
-    }
+    "SystemProperties": {}
 }
 ```
+
 *Template*
+
 ```json
 {
-    "templateType": "JsonPathContent",
+    "templateType": "CalculatedContentTemplate",
     "template": {
         "typeName": "heartrate",
-        "typeMatchExpression": "$..[?(@Body.heartRate)]",
-        "deviceIdExpression": "$.deviceId",
-        "timestampExpression": "$.endDate",
+        "typeMatchExpression": "$..[?(@heartRate)]",
+        "deviceIdExpression": "$.Properties.deviceId",
+        "timestampExpression": "$.matchedToken.endDate",
         "values": [
             {
                 "required": "true",
-                "valueExpression": "$.Body.heartRate",
+                "valueExpression": "$.matchedToken.heartRate",
                 "valueName": "hr"
             }
         ]
     }
 }
 ```
+
 ---
 
-**Blood Pressure**
+**Select and tranform incoming data**

```

Comment:
@namalu @dustinburson I believe that this example is supported by our system. Here we have two templates with the same `typeName` producing the same values, just in different ways and using different `typeMatches`. Looking at the code this should produce different `Measurements` and they'd be aggregated on the fhir transformation correctly. Is my understanding correct?